app/controllers/home_controller.rb

```diff
@@ -145,7 +145,7 @@ def activity
     # group by day: http://stackoverflow.com/questions/5970938/group-by-day-from-timestamp
     comments = comments.group('DATE(FROM_UNIXTIME(timestamp))') if Rails.env == 'production'
     comments = comments.to_a # ensure it can be serialized for caching
-    answer_comments = Comment.joins(:answer, :user)
+    answer_comments = Comment.joins(:answer, :user) # Test failing here
```

Comment:
I am not very sure what this code is doing, so I'm a bit stuck here.

Reply:
OK, here we can just remove the whole line, and lines 148-154, since those are all just collecting comments upon answers, and then also remove `answer_comments` from line 155 and 163. All comments are now just regular comments, so we don't need any of this! Thanks!